src/guides/v2.4/install-gde/system-requirements.md

```diff
@@ -1,17 +1,163 @@
 ---
 group: installation-guide
-title: System 2.4.x requirements
+title: Magento 2.4 system requirements
 functional_areas:
   - Install
   - System
   - Setup
 ---
```

Comment:
You need to add a redirect for the 2.4 version of the `system-requirements-tech.html` file you removed just like 2.3.

```
redirect_from:
  - /guides/v2.4/install-gde/system-requirements-tech.html
```

Reply:
Done